### ISSUE TITLE
Begrep Mulighet for å støtte avsnitt og kulepunkter i enkelte felt? Markdown etc?

### ISSUE THREAD

**Issue by jeffreiffers** [Informasjonsforvaltning/behov#877]

**Comment by KjerstiSteien** [comment]
@NilsOveTen @hegeaal klarer vi støtte dette behovet? Må i så fall støttes av admingrensesnitt også.

**Comment by NilsOveTen** [comment]
> @NilsOveTen @hegeaal klarer vi støtte dette behovet? Må i så fall støttes av admingrensesnitt også.

Hvorfor blander vi inn admin? Det har vel ikke noe å gjøre med faktisk innhold i feltene?

Vil tippe det største problemet blir rdf-versjonen, mener på å huske at vi har fått problem med markdown elns der før

**Comment by KjerstiSteien** [comment]
Tilknyttet sak: https://github.com/Informasjonsforvaltning/behov/issues/950

**Comment by hegeaal** [comment]
Det er vel eventuelt skjemaet som må håndtere dette. Designsystemet har dog ikke noe komponent som støtter markdown så da må vi eventuelt håndtere alt rundt det selv - kan bli litt knot.

**Comment by kristinekv** [comment]
Er dette aktuelt å ta opp igjen? @EliHelen @KurtStian

**Comment by KurtStian** [comment]
Ja, jeg tenker at det kan være aktuelt å se på hva som eventuelt skal støttes og ikke støttes av Markdown i registreringsløsningen og i portalen, dersom dette prioriteres.

I dag er støtten inkonsistent:
– I registreringsløsningen støttes ikke Markdown i detaljvisning av begrep, mens noe vises i søkeresultat.
– Når begrep høstes til portalen, støttes mer Markdown både i detaljvisning og søkeresultat.

Dette gir ulik funksjonalitet og visning av samme innhold, avhengig av hvor det presenteres. En eventuell gjennomgang bør derfor ha som mål å harmonisere støtte og visning på tvers av løsningene.

**Eksempler (forskjeller i visning):**

_Deltaljvisning - registreringsløsning_
<img width="972" height="431" alt="Image" src="https://github.com/user-attachments/assets/404f70f4-25a8-4767-b965-dfbcf567c5fc" />

_Søkeresultat - registreringsløsning_
<img width="523" height="897" alt="Image" src="https://github.com/user-attachments/assets/8a3cb3c3-2869-492b-b3b7-922708601ed2" />

_Detaljvisning - portal_
<img width="678" height="1062" alt="Image" src="https://github.com/user-attachments/assets/2462d0ef-1aa3-4be1-b649-39d210f53e19" />

_Søkeresultat - portal_
<img width="591" height="878" alt="Image" src="https://github.com/user-attachments/assets/cb3b3d4a-f4ea-4047-a8ea-87dd89d6c853" />
